apps/bot/commands/slash/get-answer-count.ts

```diff
@@ -0,0 +1,69 @@
+import { SlashCommandBuilder } from 'discord.js'
+import { SlashCommand } from '../types.js'
+import { getCorrectAnswersCount } from '../../db/actions/users.js'
+import { isUserProfilePublic } from '../../utils.js'
+export const command: SlashCommand = {
```

Comment:
Once again, consistent code styles:

```suggestion
import { isUserProfilePublic } from '../../utils.js'

export const command: SlashCommand = {
```